zebedee-cms/src/main/java/com/github/onsdigital/zebedee/dataset/api/exception/UnexpectedResponseException.java

```diff
@@ -0,0 +1,8 @@
+package com.github.onsdigital.zebedee.dataset.api.exception;
```

Comment:
I am aware I am being picky here & it is a minor thing but I wonder if its worth creating an abstract DatasetAPIException which these extend.
Being able to treat objects polymorphically is always useful. For example if we needed to write some sort of catch all error handler being able to group these and treat them generically as datasetAPI exceptions that might be quite useful.